.github/workflows/aks/aks-auto-shutdown.yaml

```diff
@@ -18,6 +18,6 @@ jobs:
           tenant-id: 531ff96d-0ae9-462a-8d2d-bec7c0b42082 # HMCTS.NET
           allow-no-subscriptions: true
       - name: AKS Auto Shutdown
-        run: ./scripts/aks-auto-shutdown.sh
+        run: ../scripts/auto-shutdown.sh
```

Comment:
@narinder17 looks like the file path is referencing auto-shutdown.sh when it should be referencing aks-auto-shutdown.sh

Reply:
will update